index.html

```diff
@@ -329,24 +338,42 @@ <h2>Relocated node connection</h2>
         </ul>
     </p>
   </div>
-  <button class="collapsible">is_valid_waited_connection</button>
-  <div class="content">
-    <p>Return true if the given message_src of the current RPC is a one of our nodes and is in State::WaitingConnectionInfo state.
-    </p>
-  </div>
   <button class="collapsible">shorter_prefix_section</button>
   <div class="content">
     <p>If we know of a section that has a shorter prefix than ours, we prefer for them to receive this incoming node rather than ourselves as it will help keep the Network's sections tree balanced.<br/>
     This shorter_prefix_section is a function that will return None if we are the shortest of any section we know, Some if there is a better candidate.<br/>
     If it is Some, we will relocate the new node to them instead of completing the relocation to our own section.
     </p>
   </div>
+  <button class="collapsible">update_to_node</button>
+  <div class="content">
+    <p>Convert the candidate node at the target interval address to a node using the new_public_id from the given CandidateInfo.
+       Update the state of the node with the given state.
+    </p>
+  </div>
   <button class="collapsible">State::RelocatingHop</button>
   <div class="content">
     <p>A nodes state indicating that it is relocated without ageing further.<br/>
        Also returned as part of waiting_proofing_or_hop().
     </p>
   </div>
+  <button class="collapsible">CANDIDATES</button>
+  <div class="content">
+    <p>Collection of candidate that we will purge if they are still in State::WaitingCandidateInfo next time Parsec::CheckRelocatedNodeConnection is consensused.
+    </p>
+  </div>
+  <button class="collapsible">CANDIDATES_INFO</button>
+  <div class="content">
+    <p>Collection of CandidateInfo keyed by CandidateInfo.new_public_id.<br/>
+    For all items, only keep if is_valid_waited_info(info)==true.
+    </p>
+  </div>
+  <button class="collapsible">CANDIDATES_VOTED</button>
```

Comment:
My understanding is that CANDIDATES_VOTED is always exactly a subset of CANDIDATES_INFO.
Maybe it would be more elegant to simply have CANDIDATE_INFO, but store a state for each candidate: "VotedFor", which could be an enum. It feels like we're storing duplicate info here.

If you disagree with this suggestion for a refactoring, I'm OK to merge as is, but consider it :)

Reply:
That was my original thinking, but it made things more confusing.